README.md

```diff
@@ -146,3 +146,6 @@ To access PhpMyAdmin, run `docker ps` to get its dynamic port.
 * Username: root
 * Password: root
 
+### SOLR Integration
+To enable SOLR, uncomment corresponding lines from docker-compose.yml file. Current version is 7.6.0, to use a different version change the SOLR_URL value with the preferred release version in the SOLR Dockerfile. Releases are available in https://archive.apache.org/dist/lucene/solr/. SOLR can be accessed in [local-domain]:8983/solr/#/ ( e.g. http://192.168.99.100:8983/solr/#/ )
```

Comment:
The image included is the just "solr" in the docker-compose file, so this will include the latest SOLR. At the moment it's 7.6.0, but this will change when a newer SOLR arrives. Could we just use the same sentence as in the Nginx branch? It would also keep things consistent between the two, see https://github.com/expresso-php/expresso-php/pull/66/files#diff-04c6e90faac2675aa89e2176d2eec7d8R151